hydra_base/util/hdb.py

```diff
@@ -136,7 +137,7 @@ def login_user(username, password):
     except NoResultFound:
         raise HydraError(username)
 
-    if get_remaining_login_attempts(username) <= 0:
+    if get_remaining_login_attempts(username, user_id=user_i.id) <= 0:
```

Comment:
`get_remaining_login_attempts()` does not use the received kwarg `user_id`

Reply:
It uses it in the 'get_user' permission check

Reply:
This is the current `get_remaining_login_attempts()` - receives `kwargs` but doesn't use any.

https://github.com/hydraplatform/hydra-base/blob/eb16ffab014dce8c140e3e6d9c70d58f4aa3a070/hydra_base/lib/users.py#L204-L210